todo_app/data/trello_items.py

```diff
@@ -0,0 +1,85 @@
+import requests
+import os
+from todo_app.data.Item import Item
+
+
+def build_parameters(extraparams={}):
+    query = {'key': os.getenv('API_KEY'), 'token': os.getenv('TOKEN')}
+    query.update(extraparams)
+    return query
+
+
+def build_url(extrapart=""):
+    url = "https://api.trello.com/1/" + extrapart
```

Comment:
I think a better name for this function parameter would be `endpoint` rather than `extrapart`.